sycl/include/CL/sycl/intel/reduction.hpp

```diff
@@ -876,7 +872,7 @@ template <typename KernelName, typename KernelType, int Dims, class Reduction,
           bool UniformWG, typename InputT, typename OutputT>
 enable_if_t<Reduction::has_fast_reduce && !Reduction::has_fast_atomics>
 reduAuxCGFuncImpl(handler &CGH, const nd_range<Dims> &Range, size_t NWorkItems,
-                  Reduction &Redu, InputT In, OutputT Out) {
+                  Reduction &, InputT In, OutputT Out) {
   size_t NWorkGroups = Range.get_group_range().size();
   bool IsUpdateOfUserAcc =
```

Comment:
Am I right in thinking that this condition is used for both the buffer and USM cases?  If so, renaming this variable to something like `IsUpdateOfUserVar` throughout might be clearer.

Reply:
Yes, it for both - buffer and USM. I updated the name of the variable. Thank you!